dev/tasks/generate-crds

```diff
@@ -55,7 +55,9 @@ go run ./scripts/crd-tools reflow-descriptions --dir apis/config/crd/
 
 # excluded_resources are resources that are under development for a direct conversion
 # we don't modify the CRD just yet for those but will in the future
-excluded_resources=("computeforwardingrule" "bigquerydatatransferconfig")
```

Comment:
We do not want to use this flag if the resource is already Beta.  https://docs.google.com/document/d/1Az2yOr9dGHrj-IGvYEhGxjpajGmcd65mQ4OjsAQSXyE/edit?resourcekey=0-Mf91G-_QDqq6XEVzf_CnQQ&tab=t.0#heading=h.8mqswg27ngyc

Reply:
Found the instruction to resolve this!